src/release/breaking-changes/new-color-scheme-roles.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Introduce new `ColorScheme` roles for Material 3
+description: >-
+  `ColorScheme` introduces new colors roles to better align with Material Design
+  3. `ColorScheme.fromSeed` method is also updated to support the newly added
+  colors.
+---
+
+## Summary
+
+New color roles in `ColorScheme` include seven tone-based surfaces and containers, alongside twelve accent color add-ons. This update deprecates three existing color roles: `background`, `onBackground`, and `surfaceVariant`. The `ColorScheme` constructed by the updated `ColorScheme.fromSeed` method now generates different values compared to the previous version, adapting to Material Design 3's guidelines.
+
+## Background
+
+The tone-based surface colors include: `surfaceBright`, `surfaceDim`,
+`surfaceContainer`, `surfaceContainerLow`, `surfaceContainerLowest`,
+`surfaceContainerHigh`, and `surfaceContainerHighest`. These changes helps 
+deprecate the use of widgets' `surfaceTintColor`, and replace the old 
+opacity-based model which applied a tinted overlay on top of surfaces based 
+on their elevation.  
+
+The `surfaceTintColor` of widgets has been set to null and their background
+color are replaced by the new tone-based surface colors.
+
+`ColorScheme.fromSeed` has also been updated to use the latest algorithm in
+Material color utilities package. This change prevent the constructed 
+`ColorSchem` from being too bright, even if the source color looks bright and
+has a high chroma.
+
+## Migration guide
+
+The difference caused by the updated `ColorScheme.fromSeed` and the new color
+roles should be small and acceptable. However, when providing a brighter
+source color to `ColorScheme.fromSeed`, it may constructs a relatively
+darker version of `ColorScheme`. To force the output to still be bright, 
+set `schemeVariant: FromSeedVariant.fidelity` in `ColorScheme.fromSeed`.
```

Comment:
```suggestion
set `schemeVariant: FromSeedVariant.fidelity` in `ColorScheme.fromSeed`. For example:
```

Reply:
Thanks a lot for all the suggestions! Just fixed them:)!